datalad/tests/test_utils.py

```diff
@@ -1094,8 +1094,17 @@ def test_get_open_files(p):
 
     assert not get_open_files(subd)
     # if we start a process within that directory, should get informed
-    from subprocess import Popen
-    proc = Popen(['cat'], cwd=subd)
+    from subprocess import Popen, PIPE
+    from time import time
+    t0 = time()
+    proc = Popen(['python', '-c',
```

Comment:
This would fail in the same scenario reported [here](https://github.com/datalad/datalad/issues/2287#issue-303452753) (i.e., no "python", just "python3").

Reply:
Eh... Thanks. Then we should use sys.executable may be here and in the procedures magical extension handling... I will fix here